app/models/Bio.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+use Carbon\Carbon;
+use Illuminate\Database\Eloquent\Collection;
+
+class Bio extends UuidBase
+{
+    protected $table = 'bios';
+
+    protected $guarded = array(
+        'id'
+    );
```

Comment:
Better to _just_ use `$fillable` I think. I dunno what the precedence rules are between the two and trying to keep the sum of them as a complete representation of the database schema is a losing battle. Also 5.3 arrays BURN IN HELL.